src/containers/project.py

```diff
@@ -0,0 +1,35 @@
+from typing import List, Dict
+
+from .project_issue import ProjectIssue
+
+
+class Project:
+    def __init__(self):
+        self.id: str = ""
+        self.number: int = 0
+        self.title: str = ""
+        self.organizationName: str = ""
+        self.repositoriesFromConfig: List[str] = []
```

Comment:
This could use Repository class.

Reply:
Containers folder is removed. Current project is supporting singleton class called GithubManager, which solves many of these issues and comments. All containers comments, can be marked as solved.